app/qml/Status/Application/MainView/MainView.qml

```diff
@@ -0,0 +1,35 @@
+import QtQuick
+import QtQuick.Layouts
+import QtQuick.Controls
+
+/// Responsible for setup of user workflows after onboarding
+Item {
+    id: root
+
+    /// Emited when everything is loaded and UX ready
+    signal ready()
+
+    Component.onCompleted: root.ready()
+
+    implicitWidth: mainLayout.implicitWidth
+    implicitHeight: mainLayout.implicitHeight
+
+    ColumnLayout {
+        id: mainLayout
+
+        anchors.fill: parent
+
+        RowLayout {}
```

Comment:
empty `RowLayout`?

Reply:
As spacers to center TODO, will be removed in the [next PR](https://github.com/status-im/status-desktop/pull/5952)